docker/README.md

```diff
@@ -0,0 +1,49 @@
+## Building ##
+
+### Pre-Requisites ###
+Both
+[`sbt`](http://www.scala-sbt.org/0.13/tutorial/Installing-sbt-on-Linux.html)
+and a
+[JDK](http://www.webupd8.org/2012/09/install-oracle-java-8-in-ubuntu-via-ppa.html)
+(oracle-jdk-8 recommended)
+must be installed to build.
+
+To build the image, run:
+```
+cd .. # Change to project root.
+sbt assembly
+cp target/scala-2.11/snapshotter-assembly-0.0.1-SNAPSHOT.jar docker/snapshotter-assembly.jar
```

Comment:
This assumes that the version number is (and will stay) `0.0.1-SNAPSHOT`.

Consider:
`cp target/scala-2.11/snapshotter-assembly-*.jar docker/snapshotter-assembly.jar `